sql/core/src/test/resources/sql-tests/inputs/pivot.sql

```diff
@@ -287,3 +287,13 @@ PIVOT (
   sum(earnings)
   FOR (course, m) IN (('dotNET', map('1', 1)), ('Java', map('2', 2)))
 );
+
+-- grouping columns output in the same order as input
+SELECT * FROM (
+  SELECT course, earnings, "a" as a, "z" as z, "b" as b, "y" as y, "c" as c, "x" as x, "d" as d, "w" as w
```

Comment:
Is it necessary to have so many columns? Probably the test case could be shorter?

Reply:
It wouldn't hurt anyway I think...